src/style.css

```diff
@@ -1 +1,89 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf');
+  font-weight: bold;
+}
+
+:root {
+  --text-font: Avenir, sans-serif;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.input__large {
+  margin: 20px 0;
+  box-sizing: border-box;
+  border-radius: 4px;
+  color: black;
+}
+
+.input__small {
+  box-sizing: border-box;
+  border-radius: 4px;
+  color: black;
+}
+
+.input__icon {
+  width: 19px;
+  height: 19px;
+  position: absolute;
+}
+
+.input__large__item {
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  background: url(images/Search.svg) no-repeat;
+  background-position:
+    25px 48%,
+    center;
+  background-size: 19px 19px;
+  border: 1px solid #e1e7ed;
+  font-size: 16px;
+  font-weight: 300;
+  font-family: var(--text-font);
+}
+
+.input__small__item {
+  box-sizing: border-box;
+  width: 100%;
+  height: 42px;
+  background: url(images/Search.svg) no-repeat;
+  background-position: 12px 48%;
+  background-size: 11px 11px;
+  padding-left: 33px;
+  border: 1px solid #e1e7ed;
+  font-size: 14px;
+  font-weight: 300;
+  font-family: var(--text-font);
+}
+
+.input__large:hover,
+.input__small:hover {
+  box-shadow: 1px 4px 4px 1px;
```

Comment:
The `box-shadow` property is missing color values. Consider specifying the color to ensure the shadow is visible.